packages/paste-website/src/pages/components/chat-composer/index.mdx

```diff
@@ -62,122 +81,426 @@ export const getStaticProps = async () => {
 
-### Accessibility
+### ChatComposer
+
+When referring to `ChatComposer` it is the rich text area only. You can use the `ChatComposer` component only, or use it within the `ChatComposerContainer` for consistent styling across chat features.
+
+#### Accessibility
 
 Chat Composer supports a variety of aria attributes which are passed into the content editable region of the component.
 
 - If the surrounding UI includes a screen reader visible label reference the label element using `aria-labelledby`.
 - If the surrounding UI does not include a screen reader visible label, use `aria-label` to describe the input.
 - If the surrounding UI includes additional help or error text use `aria-describedby` to reference the associated element.
 
-### Lexical and plugins
+#### Lexical and plugins
 
 Chat Composer is built on top of the [Lexical](https://lexical.dev) editor. Lexical is extensible and follows a declarative approach to configuration via JSX. Developers can leverage a
 wide variety of [existing plugins](https://github.com/twilio-labs/paste/blob/main/packages/paste-libraries/lexical/src/index.tsx) via the `@twilio-paste/lexical-library` package or other
 sources. Alternatively, developers can write their own custom plugin logic. Plugins are provided to the Chat Composer via the `children` prop.
 
-#### Auto Link Plugin
+##### Auto Link Plugin
 
 Chat Composer uses a custom [`AutoLinkPlugin`](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/chat-composer/src/AutoLinkPlugin.tsx) internally
 which you can see being configured [here](https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/chat-composer/src/ChatComposer.tsx#L116) as a JSX child.
 
+### Chat Composer Component Suite
+
+The Chat Composer component suite offers a variety of components designed to enhance and enrich the chat experience. Each element plays a crucial role in maintaining a consistent and cohesive styling, ensuring a seamless user interaction. The available components include:
+
+- **ChatComposerContainer**: The primary container that houses the entire chat composer interface.
+- **ChatComposerActionGroup**: A collection of action buttons and controls, allowing users to perform various actions.
+- **ChatComposerAttachmentGroup**: Groups multiple attachments together in responsive columns.
+- **ChatComposerAttachmentCard**: A card-like component for showcasing attachment previews, making it easy for users to view details at a glance with the option to set the icon for the attachment.
+- **ChatComposerAttachmentDescription**: Provides a description or additional information about an attachment, adding context for the user.
+- **ChatComposerAttachmentLink**: Creates clickable links for attachments, facilitating easy access and interaction.
+
+#### Variants
+
+ChatComposerContainer has `default` and `contained` variants. The contained variant is used as part of the new AI experience.
+
```

Comment:
suggestion: This can be removed since the contained variant can be used for any chat experience